src/Components/Clients/index.js

```diff
@@ -4,7 +4,7 @@ import styles from './clients.module.css';
 function Clients() {
   const [clients, saveClients] = useState([]);
   useEffect(() => {
-    fetch(`${process.env.REACT_APP_API}/clients`)
+    fetch(`${process.env.REACT_APP_API}/api/clients`)
```

Comment:
Same as above. Edit your .env file and do the following:
```suggestion
    fetch(`${process.env.REACT_APP_API}/clients`)
```